week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -15,8 +15,32 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 
 function sortAges(arr) {
+  
+  let newArr = [];
+  for (i = 0; i < arr.length; i++) {
 
-}
+    if (typeof arr[i] === "number") {
+      newArr.push(arr[i])
+      }
+  }
+  
+   for (let b = 0; b < newArr.length ; b++) {
+    for(let j = 0 ; j < newArr.length-1-b; j++){ // this nested loop let the function to check the condition for all the elements of the array 
+    if (newArr[j] > newArr[j + 1]) {
+  //     // swap
+      var temp = newArr[j];
+      newArr[j] = newArr[j+1];
+      newArr[j + 1] = temp;
+    }
+   }
+  }
+  return newArr
+  // console.log(newArr)

```

Comment:
It's a good idea to remove commented code before you upload to to github :)

Reply:
Thanks Chris, I am removing them asap